### DISCUSSION TITLE
What do you use for UI components?

### DISCUSSION THREAD

**Discussion by ghost** [preactjs/preact#3232] [General]
What are the best UI component libraries for use with preact? I tried to find the answer in awesome-preact, but everything is 4-5 years old.

**Comment by emschwartz** [comment]
I'm using [TailwindUI](tailwindui.com/) (the paid UI components built with [TailwindCSS](tailwindcss.com/) by the creators of the open source project) and have found it to be quite nice. Since I'm using Preact, I appreciate that Tailwind doesn't explode my bundle size. I totally understand that not everyone is able or willing to pay for TailwindUI but if it happens to be in your budget, I'd recommend considering it.

Note: I am not affiliated with Tailwind Labs in any way. Just a contented user.

**Comment by developit** [comment]
I've been using a few UI toolkits:

- React Material Web Components: https://rmwc.io/
- Spectre.css: https://picturepan2.github.io/spectre/
- Ionic web components (https://codesandbox.io/embed/inspiring-sea-gz3b5?codemirror=1)

**Comment by laoshaw** [comment]
what about bootstrap, is there a preact-bootstrap for the bootstrap 5 release? could not find any.
excited to try out preact but it seems the UI components options are limited, or I have yet to find them.

**Comment by abhishekpurecode** [comment]
Selecting a UI component library depends on project requirements and preferences. Notable choices include Bootstrap for its comprehensive set of pre-designed components and responsive grid system. Tailwind CSS, with its utility-first approach, offers highly customizable designs. Material-UI, based on Google's Material Design, provides a consistent and visually appealing UI. Ant Design is renowned for its enterprise-level components and design principles. Ultimately, the choice depends on factors like design preferences, project scale, and community support, with each library bringing its unique strengths to cater to diverse development needs.

To know more, read [here](https://purecode.ai/blogs/tailwind-components/)

**Reply by rschristian** [reply]
This reads like a ChatGPT response and doesn't really answer the question.
